### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by hectorsector**
In this PR, you'll collaborate with your classmates on Assignment 2. Here's how:

### Step 1: Think of a project idea

Throughout the semester, we'll build a set of projects that showcase what you've learned. The first project is to build a website that showcases something you're passionate about -- this will allow you to practice the lessons on HTML, Bootstrap, and GitHub. The idea here is to collaborate with teams of people. In the comments of this Pull Request, I'd like to identify something you're passionate about that you'd enjoy building a web site for. Throw ideas out there, and work with others that have done the same to help them refine it. If others voice a similar passion to yours, then you should group into a team of 2-5 people. Even after your team has formed, invite others to join your team as long as you don't go over 5 people. There will be no more than 5 teams allowed, so no one should have to work individually.

So the steps are:
1. Read through the comments in this PR to see if someone already wants to create a site you'd be passionate about creating.
1. If someone else has voiced that passion, leave a comment letting them know you'd like to join them, and give some ideas about the direction you'd like to take the site in.
1. If someone has not voiced your passion, leave a comment letting classmates know what you'd like to build and that you're open for partners. 
1. Once you've got at least 2 people, edit this post by clicking on the ✏️ icon, and jot down your usernames for that team. Feel free to give your team an awesome name. 

- Team 1 aka YMD Marvel: 
  - @marinovp 
  - @YolieBez 
  - @dabouelhassan
  - Closed
  - Closed
- fantastic5: 
  - @marullo
  - @demetrejj
  - @jwalker139
  - @abrahamf101
  - @DLiberal
- Team 3: 
  - @Ramonsito21
  - @Suhon 
  - @rsines 
  - @PlankLongBeard 
  - @AspenDangren
- Team 4: 
  - @mcampbe11
  - @tylersullivan21
  - @BrendonGonzales
  - @aperez214
  - @kmolander
- Team 5: 
  - @yourusernamehere
  - @yourusernamehere
  - @yourusernamehere
  - @yourusernamehere
  - @yourusernamehere

### Step 2: Commit to your idea

1. On Git Kraken, checkout to the `assignment-2` branch.
1. On your computer, locate the folder that contains the class repository. There, you'll find the directory: `full/assignment-2/`. Double-click `index.html` to open it in Safari, Chrome, or Firefox. Try not to use Internet Explorer. This shows you how to test the webpage locally.
1. In the same folder, right-click on `index.html` and open it with Atom or your favorite text editor.
1. In the `index.html` file, you'll see a series of `<article>` containers with a `thumb` class and some [lorem ipsum](http://www.lipsum.com). Choose a container and replace its filler text with your team name within the `<h2>` tag, and a description of your team project within the `<p>` tag. Also replace the default image with one that describes your team. Have fun here -- design one from scratch or find one on the internet that captures your team's goals.
1. Once you've saved your changes, refresh `index.html` on your web browser to ensure your changes look how you expect.
1. Stage, commit, and push your additions. 

### Step 3: Start planning your site (planning happens this week, buildings starts in a couple of weeks)

1. Open a new issue for you and your team to discuss the direction of the site -- `@mention` your team on the issue.
1. In the open issue, talk through the contents of the site. Plan out what you'll include in terms of actual content, words on page, pages, links, etc.
1. Also talk through site design -- start looking for Bootstrap templates or other things you could use to get you the look and feel you want (some examples: [Start Bootstrap](https://startbootstrap.com), [W3Schools](http://www.w3schools.com/bootstrap/bootstrap_templates.asp), and [ShapeBootstrap](https://shapebootstrap.net/free-templates)).
1. Start keeping track of any resources that you use from the internet, you'll need to give them credit when your site goes live. 

For future reference, the finished site will need to meet the following criteria:
- Bootstrap is used.
- Web site contains images.
- Web site contains text.
- Web site contains external links.
- All of the web site's content can be accessed just by scrolling.
- Users can navigate to different sections of the webpage by clicking buttons in the navigation.
- The web site contains original content.
- The web site doesn't contain inappropriate content, as described in the Student Code of Conduct.
- The web site is published to GitHub Pages.
- The link to the web site is submitted in the class repository.

### CODE REVIEW

**Comment by dabouelhassan** [comment]
I'm a news politics junky! I know, it's sad. I was thinking a bout a site to pull tweets from POTUS and other "leaders" and news about specific hot policies that are being debated that would affect us.

**Comment by marullo** [comment]
Hey class, anybody is welcome to join our group. We are looking to share idea's for the project.

**Comment by mcampbe11** [comment]
Hi there! Is anyone passionate about adventurous video games or just games in general? I'd love to build a website based around games such as Life is Strange, Oxenfree, Inside, Skyrim, Tomb Raider, and Bioshock.

**Comment by hectorsector** [comment]
@mcampbe11 it sounds like you can join @marullo's team!

**Comment by jwalker139** [comment]
@marullo what are you thinking about doing. Where are you guys communicating at?

**Comment by marullo** [comment]
@jwalker139 there are 2 more in team2 list but I haven't received any reply from them about any ideas. Since food and travel are my dreams and passions, I was thinking about maybe a food destination page, or a website about some video game.

**Comment by marullo** [comment]
@jwalker139 I'm on slack, if you want you can create a group and add me

**Comment by DLiberal** [comment]
alright, ill be on slack. who wants to add me

**Comment by Demetrejj** [comment]
@marullo I am interested in the same things, I would love to do either topic.

**Comment by marullo** [comment]
@Demetrejj I was talking to @DLiberal and we have an idea of doing something like a historic landmarks info., what do you think. we were thinking about putting gen info about them as well as to put some info of  the culture of the place where the landmark is located

**Comment by marullo** [comment]
something like that

**Comment by Demetrejj** [comment]
@marullo awesome , sounds good to me. Let's do this.

**Comment by Abrahamf101** [comment]
@marullo I share the same passion on those two topics. Food definitely over travel but I do wish to travel all over the world soon man. I think we could agree on a topic for a website

**Comment by marullo** [comment]
@Abrahamf101 we were thinking about historic landmark info page. what do you think?

**Comment by marullo** [comment]
with info on the culture of the place the landmark is located

**Comment by Abrahamf101** [comment]
@marullo  Yea, I like that idea. Can i join the group?

**Comment by marullo** [comment]
yeah

**Comment by marullo** [comment]
@Abrahamf101 if anything just hit me up at slack

**Comment by Abrahamf101** [comment]
Ok, thanks 👍

**Comment by jwalker139** [comment]
I like your idea. I will join your group @marullo

**Comment by mcampbe11** [comment]
Still looking for one or more people interested in a website on a video game!

**Comment by aperez214** [comment]
@mcampbe11 I don't mind doing a website on videogames but just a heads up I don't really play single player games, I know about the majority of them but I play more FPS and Fighting games lol. I'm sure we can figure something out!

**Comment by BrendonGonzales** [comment]
@mcampbe11 I also don't mind doing a website on video games, it sounds like a cool concept. I'll be in your group if you don't mind.

**Comment by tylersullivan21** [comment]
@mcampbe11 I'm highly interested in a variety of video games! I'd like to join your group if you don't mind! :)

**Comment by mcampbe11** [comment]
@aperez214 @BrendonGonzales @tylersullivan21 Awesome! I'll add my name to Team 4 if you guys want to do the same. I'll also try to figure out how to create a Slack group chat for us :)

**Comment by Suhon** [comment]
Hello everyone!, I know i'm kinda joining the game a little late. I'm interested in anything but I would really like to create a sort of catalog such as this http://www.target.com/c/home-style-catalog/-/N-4sjwc or for any kind of products.

**Comment by mcampbe11** [comment]
Hey @aperez214 @BrendonGonzales @kmolander I created a group chat for us on Slack if you want to check that out!

**Comment by ramonsito21** [comment]
Late to the assignment but any open team out there that i can put my knowledge to work?

**Comment by ramonsito21** [comment]
If any room let me know ill keep checking in.

**Comment by rsines** [comment]
I'm late to the party :-) Anyone have room for me? @Marinovp @YolieBez @dabouelhassan @jaj371 mind if I join yours? I'm a political junkie myself too!

**Comment by Suhon** [comment]
@ramonsito21 Would you like to make a group with me?

**Comment by YolieBez** [comment]
@jaj371 @rsines Hey guys, we Closed our group last week. Please create a group of your own. @Suhon @ramonsito21 are looking to start a group - now you have 4 members.

**Comment by hectorsector** [comment]
Great job so far, everyone!

Could everyone please add the team member to the web page as well? A list of their handles or their first names is 🆗

File: full/assignment-2/index.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+							<h3>The Taj Mahal</h3>
+							<p>The Taj Mahal is in Agra, India. Taj Mahal means "Crown of the Palace".</p>
+						</article>
+						<article class="thumb">
+							<a href="images/fulls/02.gif" class="image"><img src="https://s-media-cache-ak0.pinimg.com/736x/b9/1c/12/b91c12b763b58b1061fcd34e6670f565.jpg" alt="" /></a>
+							<h2>YMD Marvel</h2>
+							<p>Epic Marvel Battles.</p>
+						</article>
+						<article class="thumb">
+							<a href="http://hdqwalls.com/wallpapers/god-of-war-2017-to.jpg" class="image"><img src="https://lh3.googleusercontent.com/-F7bc_T-MP2Y/WJFc53cuTYI/AAAAAAAAAOc/N3vZ4EYPUpwEGdFV6iwxdYOAgeSO-AZ4ACL0B/w663-d-h373-p-rw/TheGameaholics.jpg" alt="" /></a>
```
Comment (hectorsector) [inline full/assignment-2/index.html]: Gameaholics: it seems that the image on `googleusercontent.com` isn't loading -- is it loading for you? You can also put the image you want into the site's `images` folder, and then reference it from there.

Comment (hectorsector) [inline full/assignment-2/index.html]: ☝️ @mcampbe11 @tylersullivan21 @BrendonGonzales @aperez214 @kmolander

**Comment by hectorsector** [comment]
@cop1000/201720-full-term-students for now, the class project site is visible at https://cop1000.github.io/201720/full/assignment-2/ -- when we're done with it I'll move it a friendlier URL, but it still needs a little work.

Is anyone willing to ✋ remove extra tiles for sections that aren't being used, and willing to change some of the bottom bar that has an "about" section? Multiple people can work on it, you can contribute by adding your commits directly.

**Comment by ramonsito21** [comment]
@suhon sure

**Comment by ramonsito21** [comment]
@suhon what you have in mind?

**Comment by Suhon** [comment]
@ramonsito21 I was just thinking something simple like a catalog for a store but I'm open for any ideas you have I'm in class right now so it might take me a bit to reply

**Comment by ramonsito21** [comment]
@suhon that doesn't sound bad. How can I message you that's not here?

**Comment by Suhon** [comment]
@ramonsito21 if you want you can email me at acaicedo12@hotmail.com

**Comment by rsines** [comment]
@yoliebez my apologies I already added myself to the group like the instructions said to do. I didn't realize the group was "closed" as there were not 5 people yet and I thought that part was due today?

**Comment by YolieBez** [comment]
@rsines Everybody Had to Discuss it first, before adding to the group. It is said to have two people min and 5 max, it is not required to have 5 people in the group. There were other classmates, who though the same thing, but we already closed our Group 1. We are open to help each other, so please find yourself another group. Thank you. 
Sincerely,
YolieBez

**Comment by YolieBez** [comment]
@Suhon @ramonsito21 @jaj371 @rsines here are people who were looking to start a group.

**Comment by YolieBez** [comment]
@hectorsector I am willing to look into it and see if I can help out. ^_^

**Comment by rsines** [comment]
@yoliebez not a problem don't worry about it @jaj371 @Suhon @ramonsito21 are yall still looking for a group? I'm cool with the catalogue idea. Or maybe something on nutrition?

**Comment by YolieBez** [comment]
@rsines Thank you, we appreciate your understanding. ^_^ .

**Comment by ramonsito21** [comment]
@Suhon i messaged you on slack.

**Comment by Suhon** [comment]
@rsines Yeah you're welcome to join 😄 do you have slack,discord,skype or anything we can use to chat on?

**Comment by rsines** [comment]
@suhon Great! I have slack and Skype, not discord but I can set it up if needed. What do you prefer?

**Comment by Suhon** [comment]
@rsines Slack is fine since we already have it

**Comment by dabouelhassan** [comment]
Just following up on what @YolieBez said about late members .. our Group 1 has been closed While the professor still recording the Tutorial for assignment 2.  Obviously small groups tend to collaborate efficiently ,and ours already laid the ground work and a vision forward, but ultimately we are all here to  learn and collaborate with each-other as my other colleagues @YolieBez and @Marinovp  already mentioned

**Comment by PlankLongBeard** [comment]
Hey all, I've been looking over the comments and I've been "out of the office" for the past few days on some Family Care needs. I'd like to do a page on Space and it's awesomeness. If that's no bueno, then just let me know which group I can kindly join or be part of, so I can get into the swing of things. Thanks :)

**Comment by Suhon** [comment]
@PlankLongBeard Hey man, would you want to join our team? We're still undecided on what to do and I like the space idea better than mine.

**Comment by PlankLongBeard** [comment]
That would be awesome @suhon! Did you pick a Team Name yet? If not, I think Space Crusaders or Team Nebula sounds great!

**Comment by Suhon** [comment]
@PlankLongBeard Alright we'll add you

**Comment by rsines** [comment]
@planklongbeard we added you to slack

**Comment by PlankLongBeard** [comment]
Awesome @rsines and @Suhon! I look forward to working with all of you :)

**Comment by hectorsector** [comment]
> anyone knows why my commits are showing as unknown ?
> I tried to reauthenticate.. done it through terminal git push.. but still getting "unknown"through slack

@dabouelhassan did you figure this out? If not, open a new issue with a description and we can work through it.

**Comment by hectorsector** [comment]
If anyone is still struggling with finding a group, please either let us know here, or feel free to message me privately and I can help you find a home ❤️ @cop1000/201720-full-term-students

File: full/assignment-2/index.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+						<article class="thumb">
+							<a href="images/fulls/02.gif" class="image"><img src="https://s-media-cache-ak0.pinimg.com/736x/b9/1c/12/b91c12b763b58b1061fcd34e6670f565.jpg" alt="" /></a>
+							<!--<h2>YMD Marvel</h2>
+							<p>Epic Marvel Battles.</p>-->
+							<div class="bottomright"><ul> Team: Yolie, Marino, Driss</ul></div>
+						
+
+						</article>
+						<article class="thumb">
+							<a href="http://hdqwalls.com/wallpapers/god-of-war-2017-to.jpg" class="image"><img src="https://lh3.googleusercontent.com/-F7bc_T-MP2Y/WJFc53cuTYI/AAAAAAAAAOc/N3vZ4EYPUpwEGdFV6iwxdYOAgeSO-AZ4ACL0B/w663-d-h373-p-rw/TheGameaholics.jpg" alt="" /></a>
```
Comment (hectorsector) [inline full/assignment-2/index.html]: Thanks for that @cop1000/gameaholics-anonymous! So now that you've got the image properly uploaded, you've got to tell the `index.html` file where to find it. Currently, your `index` file is still looking for the file at `googleusercontent.com`, so update this reference with your newly uploaded image. Check out how some of the other teams did it to give you an idea of what you could do.

**Comment by hectorsector** [comment]
@cop1000/201720-full-term-students I've now merged this branch, which means I've incorporated the changes into the core `201720` project. You can see our final published site at https://cop1000.github.io/201720/ ✨ ❤️  Fantastic work 👏 

This site is a work in progress, so you can continue making edits or improvements to it, especially if you'd like to hone your skills on a site that's now "live". If you'd like to continue making changes to the site, this is what you should do:
1. On Git Kraken, check out to the `master` branch.
1. On `master`, run a `Pull` operation to update your `master` branch with the one on the remote.
1. Create a new branch.
1. Make a series of commits on your new branch with your changes.
1. Test your changes by opening up the `index` file on your web browser and ensuring everything looks ✨.
1. Push the branch up to GitHub.
1. Open a Pull Request describing the changes, ping some people in the PR so you can get 👀 on your work.
1. When you've gotten 👀 on your work, you can merge the branch, but make sure at least 1 or 2 people have reviewed it before you merge. 
1. When you merge the branch, in a minute or two you'll see your changes [live on the site](https://cop1000.github.io/201720/).

**Pull request merged**